src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunGenericHelperNode.cs

```diff
@@ -49,6 +49,8 @@ private static GenericLookupResult GetLookupSignature(NodeFactory factory, Ready
                     return factory.GenericLookup.MethodDictionary((MethodDesc)target);
                 case ReadyToRunHelperId.VirtualCall:
                     return factory.GenericLookup.VirtualCall((MethodDesc)target);
+                case ReadyToRunHelperId.VirtualDispatchCell:
+                    return factory.GenericLookup.VirtualMethodAddress((MethodDesc)target);
                 case ReadyToRunHelperId.ResolveVirtualFunction:
```

Comment:
I do not see why we need both VirtualDispatchCell and ResolveVirtualFunction ... they look the same to me. But I guess there is a good reason since you have kept it.